.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature Request]"
+labels: 'enhancement'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
I'd change `Ex.` to `E.g.` just b/c it's technically more correct

Reply:
I'd suggest not framing this in a negative way. Instead of suggesting that the user is starting from a place of frustration with our product, I'd instead suggest something like "I'd love to see `x` capability integrated into Pinecone b/c of `y`"